TODO.md

```diff
@@ -0,0 +1,46 @@
+# TODO
+
+## Basic Components
```

Comment:
I think we should implement a new shadow render

Reply:
We can implement it in the next few versions. Do you have any good idea?

Reply:
Fork from Modifier.shadow，and change some attributes ？

Reply:
`Modifier.shadow()` is based on `graphicsLayer()`, which is in `ui.graphics` package. It's not very easy to do that. Fluent Design requires custom shadow radius, spread and color. I've seen some CSS-like implementations on Slack and StackOverflow that use native Skia API, may not work on Android platform.

Reply:
See [Issue 160665122#106](https://issuetracker.google.com/issues/160665122#comment106). Can't understand why doesn't Google want to implement that.

Reply:
We can merge it into master so people can use this lib in compose 1.5. Implement other features later.